README.md

```diff
@@ -34,14 +34,39 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. What is the difference between an adaptive website and a fully responsive website?
 
+
+ The important difference is that responsive design is fluid and adapts the size of the screen no matter the device. 
+ Resposnive design uses CSS media to change the style, to resize the height of the page , the  width of the screen.
+ Adaptive on the other hand uses static layouts that needs to depend on the screen size of the device.
+
+
 2. Describe what it means to be mobile first vs desktop first.
 
+
+Mobile first is the ability to implement responsive design to tell Web Sites to adapt to whatever device a person
+is using to see this Web Site.
+
+Desktop First means to hide an wrap elements to support in the mean time original features as possible.
+
+
```

Comment:
Nice attempt to answer this question. This could be better and I will enjoin you to check [this article](https://www.h3xed.com/web-development/responsive-css-mobile-first-vs-desktop-first-media-queries) out to understand the difference between desktop first as opposed to  mobile-first